CODE_OF_CONDUCT.md

```diff
@@ -1,10 +1,8 @@
 # Code of Conduct
 
-> This is the ***TENTATIVE*** Code of Conduct of the Mobile Development Club at University of Washington.
-
 ## 1. Purpose
 
-A primary goal of Mobile Development Club is to be inclusive to the largest number of community members, with the most varied and diverse backgrounds possible. As such, we are committed to providing a friendly, safe and welcoming environment for all, regardless of 
+A primary goal of the ***Mobile Development Club** at University of Washington* is to be inclusive to the largest number of community members, with the most varied and diverse backgrounds possible. As such, we are committed to providing a friendly, safe and welcoming environment for all, regardless of 
```

Comment:
Maybe *at **the** University of Washington*? Other than that, this looks good to me!

Reply:
```suggestion
A primary goal of the ***Mobile Development Club** at the University of Washington* is to be inclusive to the largest number of community members, with the most varied and diverse backgrounds possible. As such, we are committed to providing a friendly, safe and welcoming environment for all, regardless of 
```